### ISSUE TITLE
Demonstrate Abscal on Simulated Data

### ISSUE THREAD

**Issue by jsdillon** [HERA-Team/hera_cal#485]
We need try absolutely calibrating a day against a simulation if we want to use this for RTP.

As currently written, the model doesn't need to have all unique baselines, but it needs to include all antennas and all redundant baselines (these can be copies by assuming idealized antenna positions) for the full band and 0 to 12 hours of LST. Does that seem reasonable @zacharymartinot?

**Comment by nkern** [comment]
Not to be nitpicky, but I keep encountering this issue with vocabulary and I want to be clear so we are all on the same page. __We always calibrate against simulations at some point.__ IDR2.1 and IDR2.2 are calibrated fundamentally against a simulation: its just a CASA simulation. So, I think the appropriate way to phrase and think about this is: we need to demonstrate abscal against a simulation with _more_ components (diffuse, point and extended sources), and against some of our home-grown visibility simulators. Lmk if there is any confusion on this

**Comment by jsdillon** [comment]
Fair point, @nkern. In particular, I want to look at using the "post-redcal-abscal" algorithm against purely simulated visibilities, rather than externally calibrated visibilities (that themselves have been calibrated against visibilities created by CASA with a sky model).

**Comment by jsdillon** [comment]
@zacharymartinot We need to discuss (ideally in the next analysis telecon) what is considered "success" on this so we can go on to making the H3C model (#538).

**Comment by jsdillon** [comment]
From 10/9/19 Analysis Meeting:

To close this issue, Zach to put together a memo explaining:

- Which sources/catalogs to use
- Which baselines to calibrate against
- Which LSTs to blacklist
